components-core/src/main/java/com/adyen/checkout/components/analytics/AnalyticEvent.java

```diff
@@ -100,7 +99,7 @@ public static AnalyticEvent create(@NonNull Context context, @NonNull Flavor fla
             default:
                 throw new CheckoutException("Unexpected flavor - " + flavor.name());
         }
-        return new AnalyticEvent(context.getPackageName(), flavorName, components, LocaleUtil.toLanguageTag(locale));
+        return new AnalyticEvent(context.getPackageName(), flavorName, components, locale.toString());
```

Comment:
Have you tested this in older API versions? Maybe do a quick test on API 21, because I think there was some limitation.

Reply:
tested, works fine 👍